js/graviex.js

```diff
@@ -0,0 +1,1012 @@
+        //   "executed_volume": "0.0",
+        //   "trades_count": 0,
+        //   "trades": [],
+        //   "strategy": null
+        // }
+        const order = this.parseOrder (response);
+        const status = order['status'];
+        if (status === 'closed' || status === 'canceled') {
+            throw new OrderNotFound (this.id + ' ' + this.json (order));
+        }
+        return order;
+    }
+
+    async clearOrder (side = undefined, symbol = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+        };
+        if (side !== undefined) {
+            request['side'] = side;
+        }
+        const response = await this.privatePostOrdersClear (this.extend (request, params));
+        const order = this.parseOrder (response);
+        const status = order['status'];
+        if (status === 'closed' || status === 'canceled') {
+            throw new OrderNotFound (this.id + ' ' + this.json (order));
+        }
+        return order;
+    }
+
+    async createOrders (symbol, orders, params = {}) {
+        if (symbol === undefined) {
+            throw new ArgumentsRequired (this.id + ' createOrders() requires a symbol argument');
+        }
+        await this.loadMarkets ();
+        const market = this.market (symbol);
+        const request = {
+            'market': market['id'],
+            'orders': orders,
+        };
+        // orders: [{"side":"buy", "volume":.2, "price":1001}, {"side":"sell", "volume":0.2, "price":1002}]
+        const response = await this.privatePostOrdersMulti (this.extend (request, params));
+        const data = response['data'];
+        return this.parseOrders (data);
+    }
+
+    async fetchOrdersHistory (symbol = undefined, since = undefined, limit = undefined, params = {}) {
+        await this.loadMarkets ();
+        const request = {
+            'order_by': 'desc',
+        };
+        let market = undefined;
+        if (symbol !== undefined) {
+            request['market'] = this.marketId (symbol);
+            market = this.market (symbol);
+        }
+        if (limit !== undefined) {
+            request['limit'] = limit; // default: 500
+        }
+        const response = await this.privateGetOrdersHistory (this.extend (request, params));
+        // [
+        //   {
+        //     "id": 287792646,
+        //     "at": 1643956089,
+        //     "side": "sell",
+        //     "ord_type": "limit",
+        //     "price": "0.141",
+        //     "avg_price": "0.0",
+        //     "state": "cancel",
+        //     "market": "ltcbtc",
+        //     "created_at": "2022-02-04T09:28:09+03:00",
+        //     "volume": "0.141",
+        //     "remaining_volume": "0.141",
+        //     "executed_volume": "0.0",
+        //     "trades_count": 0,
+        //     "strategy": null
+        //   }
+        // ]
+        return this.parseOrders (response, market, since, limit);
+    }
+
+    nonce () {
```

Comment:
you don't need this method, just do `const nonce = this.milliseconds()` whenever you need a nonce 😛